samples/bluetooth/central_hids/README.rst

```diff
@@ -93,10 +93,13 @@ Testing with another board
 
 
 #. Press Button 1 and Button 3 on the Central HIDS board and observe that LED 1 on the keyboard board changes its state.
```

Comment:
In all these steps we say "press Button x and Button y on the board (...)".
It seems slightly confusing to me, my first understanding is to press them both at the same time while they should clearly be pressed consecutively. I will try to find better wording for this.

Reply:
Yes, it may be worth re-phrasing.

Reply:
In this case these Buttons can be pressed consecutively in any order as they have the same effect from the user point of view - they just use different GATT procedures to update the characteristic.